CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🐞 Fixed a bug that caused `vast import` processes to produce `'caf'` table
```

Comment:
```suggestion
- 🐞 Fixed a bug that caused `vast import` processes to produce `'default'` table
```

We haven't renamed that (yet).

Reply:
Yeah, I changed it, even though the sentence is pretty confusing this way. Let's not get hung up on this though.